packages/layout/src/FzLayout.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <div class="fz-layout w-dvw h-dvh grid gap-10 p-16" :class="layoutClass">
+        <template v-if="['twoColumns', 'leftShoulder', 'rightShoulderNavbar', 'squares'].includes(props.layout)">
+            <slot></slot>
+        </template>
+        <template v-if="props.layout === 'leftShoulderNavbar'">
+            <div class="col-span-2">
+                <slot name="navbar"></slot>
+            </div>
+            <div class="col-span-2">
+                <slot name="header"></slot>
+            </div>
+            <div class="col-span-2 lg:col-span-1">
+                <slot name="left-shoulder"></slot>
+            </div>
+            <div class="col-span-2 lg:col-span-1">
+                <slot></slot>
+            </div>
+        </template>
+        <template v-if="props.layout === 'rightShoulderNavbar'">
+            <div class="col-span-2">
+                <slot name="navbar"></slot>
+            </div>
+            <div class="col-span-2">
+                <slot name="header"></slot>
+            </div>
+            <div class="col-span-2 lg:col-span-1">
+                <slot></slot>
+            </div>
+            <div class="col-span-2 lg:col-span-1">
+                <slot name="right-shoulder"></slot>
+            </div>
```

Comment:
I would rather have a single template for each layout, it's more redundant but makes it more clear and easier to maintain
There's also two default slots for `rightShoulderNavbar`

Reply:
I'll fix the double slot. As per the single template, I actually prefer this way at least for "default slot - only" templates.